storage/src/main/java/fk/prof/storage/AsyncStorage.java

```diff
@@ -15,7 +15,7 @@
      * @param path
      * @param content
      */
-    void storeAsync(String path, InputStream content);
+    void storeAsync(String path, InputStream content, long length);
```

Comment:
If in future we need a custom callback for every invocation, we can add it later.